javascript/organizing-js/tic-tac-toe-project.md

```diff
@@ -145,7 +147,7 @@ To add your solution to the list below, edit this [file](https://github.com/TheO
 - [Harry Coburn's Solution](https://github.com/mattibun/odin-tic-tac-toe) - [View in Browser](https://mattibun.github.io/odin-tic-tac-toe)
 - [Martink-rsa's Solution](https://github.com/martink-rsa/tic_tac_toe/) - [View in Browser](https://martink-rsa.github.io/tic_tac_toe/)
 - [Joe Lee's Solution](https://github.com/joedravarol/tic_tac_toe) - [View in Browser](https://joedravarol.github.io/tic_tac_toe/)
-- [Brendaneus' Solution](https://theodinprojects.live/courses/javascript/projects/tic-tac-toe-(javascript))
+- [Brendaneus' Solution](<https://theodinprojects.live/courses/javascript/projects/tic-tac-toe-(javascript)>)
```

Comment:
It seems you have a formatter on, please turn it off and re-submit your solution, thanks!